labcontrol/db/process.py

```diff
@@ -1087,9 +1087,11 @@ def _calculate_norm_vol(dna_concs, ng=5, min_vol=2.5, max_vol=3500,
         ng : float, optional
             The amount of DNA to pool (ng). Default: 5
         min_vol : float, optional
-            The minimum volume to pool (nL). Default: 2.5
-        max_vol : float, optional
-            The maximum volume to pool (nL). Default: 3500
+            The minimum volume to pool (nL). Default: 25
+       max_vol : float, optional
```

Comment:
I think the indent is missing a space here

Reply:
Good eye!  Thanks :)